test/sharness/t0110-gateway.sh

```diff
@@ -116,6 +116,11 @@ test_expect_success "output looks good" '
 	grep "Content-Type: text/html" indexout
 '
 
+test_expect_success "HEAD 'index.html' has no content" '
+	curl -X HEAD --max-time 1 http://127.0.0.1:$port/ipfs/$INDEXHASH/ > output;
```

Comment:
should have an `&&` between these. otherwise this will pass if the curl command fails